src/blocks/SeamMeme.tsx

```diff
@@ -0,0 +1,19 @@
+import { BlockModel, ComposerComponentProps, FeedComponentProps } from "./types";
+import AppComposer from "./utils/SeamMeme/components/AppComposer";
+import AppFeed from "./utils/SeamMeme/components/AppFeed";
+
+export const SeamComposerComponent = ({ model, done }: ComposerComponentProps) => {
```

Comment:
Same with renaming the Miniapp, renaming these composer and feed components is necessary so that these names aren't as generic.